AutomatedTesting/Gem/PythonTests/prefab/PrefabLevel_BasicWorkflow.py

```diff
@@ -0,0 +1,59 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+# fmt:off
+class Tests():
+    create_new_entity = ("Entity: 'CreateNewEntity' passed", "Entity: 'CreateNewEntity' failed")
+    create_prefab = ("Prefab: 'CreatePrefab' passed", "Prefab: 'CreatePrefab' failed")
+    instantiate_prefab = ("Prefab: 'InstantiatePrefab' passed", "Prefab: 'InstantiatePrefab' failed")
+
+
+# fmt:on
+
+def PrefabLevel_BasicWorkflow():
+    """
+    Opens the level that contains 2 entities, "EmptyEntity" and "EntityWithPxCollider".
+    This test makes sure that both entities exist after opening the level and that:
+    - EmptyEntity is at Position: (10, 20, 30)
+    - EntityWithPxCollider has a PhysXCollider component
```

Comment:
Looks like this was maybe copy/pasted from somewhere else, and isn't valid for this test?

Reply:
Fixed.